webpack/stories/index.js

```diff
@@ -112,3 +114,29 @@ storiesOf('Power Status', module)
       />
     )
   );
+
+function dismiss() {
+  console.log('callback executed');
```

Comment:
It makes more sense to me to show this somehow without requiring the console to be open - maybe an alert?